tests/unit/workspace_access/test_generic.py

```diff
@@ -541,3 +548,54 @@ def test_workspace_snapshot():
 
     assert len(result_set) == 2
     assert result_set[0] == WorkspaceObjectInfo("NOTEBOOK", "123", "/rootobj/notebook1", "PYTHON")
+
+
+def test_eligibles_assets_without_owner_should_be_ignored():
+    ws = MagicMock()
+    ws.clusters.list.return_value = [
+        ClusterDetails(
+            cluster_id="1234",
+        )
+    ]
+
+    def perms(object_type: str, object_id: str):
+        if object_type == "clusters":
+            return ObjectPermissions(
+                object_id=object_id,
+                object_type=object_type,
+                access_control_list=[
+                    AccessControlResponse(
+                        group_name="de", all_permissions=[Permission(permission_level=PermissionLevel.CAN_USE)]
+                    )
+                ],
+            )
+        elif object_type == "pipelines":
+            return ObjectPermissions(
+                object_id=object_id,
+                object_type=object_type,
+                access_control_list=[
+                    AccessControlResponse(
+                        group_name="de", all_permissions=[Permission(permission_level=PermissionLevel.CAN_USE)]
+                    )
+                ],
+            )
+
+    ws.permissions.get.side_effect = perms
+
+    sup = GenericPermissionsSupport(
+        ws=ws,
+        listings=[
+            Listing(ws.clusters.list, "cluster_id", "clusters"),
+            Listing(ws.pipelines.list, "pipeline_id", "pipelines"),
+        ],
+    )
+    executables = list(sup.get_crawler_tasks())
+    tasks = [task() for task in executables]
+    assert len(tasks) == 1
+    assert tasks[0] == Permissions(
+        object_id="1234",
+        object_type="clusters",
+        raw='{"access_control_list": [{"all_permissions": ['
```

Comment:
don't test for serialized JSON, because the order of fields may change between invocations.